### DISCUSSION TITLE
Docker issue

### DISCUSSION THREAD

**Discussion by ayu-shah0912** [anasty17/mirror-leech-telegram-bot#1333] [Q&A]
app_1  | fatal: unable to access 'https://github.com/anasty17/mirror-leech-telegram-bot/': Failed to connect to github.com port 443: Connection timed out
app_1  | 2023-01-13 10:54:36,277 - root - ERROR - Something went wrong while updating, check UPSTREAM_REPO if valid or not!
app_1  | 2023-01-13 10:54:37,118 - root - INFO - Creating client from BOT_TOKEN
app_1  | 2023-01-13 10:54:37,119 - root - WARNING - MEGA Credentials not provided!
app_1  | [2023-01-13 10:54:37 +0000] [24] [INFO] Starting gunicorn 20.1.0
app_1  | [2023-01-13 10:54:37 +0000] [24] [INFO] Listening at: http://0.0.0.0:80 (24)
app_1  | [2023-01-13 10:54:37 +0000] [24] [INFO] Using worker: sync
app_1  | [2023-01-13 10:54:37 +0000] [49] [INFO] Booting worker with pid: 49

I keep getting this while trying to run the bot

**Comment by anasty17** [comment]
UPSTREAM_REPO is wrong 
READ THE ERROR!
